src/actions/registrationActions.js

```diff
@@ -0,0 +1,55 @@
+import toastr from "../helpers/toastrConfig";
+
+import { REGISTER_REQUEST, REGISTER_SUCCESS, REGISTER_FAILURE } from "./actionTypes";
+import instance from "../api/axiosConfig";
+
+/* eslint-disable import/prefer-default-export */
+export const registerRequest = () => ({
+  type: REGISTER_REQUEST,
+  isFetching: true
```

Comment:
you don't have to have this as part of the action  you only need to change it on the reducer